index.html

```diff
@@ -10591,7 +10591,7 @@ <h3>Webhook Event Example</h3>
             <code>subscriptionID</code> is returned by the subscription. The <i>TemperatureSensor</i> will then
             periodically POST to this callback URI with a payload defined by <code>data</code>. To unsubscribe, the
             <a>Consumer</a> has to submit the <code>unsubscribeevent</code> form with the <code>subscriptionID</code> as
-            described in <code>cancellation</code>. Alternatively, <code>uriVariables</code> approache can be used that
+            described in <code>cancellation</code>. Alternatively, the <code>uriVariables</code> approach can be used that
             informs the <a>Consumer</a> to include the <code>subscriptionID</code> string into the URI that have to be
             called with the delete method (see tab 'With uriVariables'). In such setup, the
```

Comment:
```suggestion
            described in <code>cancellation</code>. Alternatively, a <code>uriVariables</code> approach, that
            informs the <a>Consumer</a> to include the <code>subscriptionID</code> string in the URIs that have to be
            called with the delete method (see tab 'With uriVariables'), can be used. In such setup, the
```

Align index.html changes with index.template.html